app/assets/stylesheets/admin/_govspeak-help.scss

```diff
@@ -11,5 +11,10 @@
 
   pre {
     white-space: pre-wrap;
+    padding: govuk-spacing(2);
+    background-color: govuk-colour("light-grey");
+    word-break: normal;
+    word-wrap: normal;
+    overflow: auto;
```

Comment:
Yes, good point. Seems we have a new file for this purpose that will be rationalised at a later date. For now specific styles like this are in `app/assets/stylesheets/admin/views/_whats-new.scss`.

Reply:
Hey @davidtrussler - I think I may have not been clear when I said this is where we're storing view specific CSS.

What I meant was you should create a separate file like the `what-new.scss` file. However, looks more like a component style, therefore, I would recommend we put these in component folder in admin - `app/assets/stylesheets/admin/components/_govspeak-help.scss`.

I would also create CSS directly for the block you're trying to modify so `.app-c-govspeak_help__pre` rather than `.app-c-govspeak_help pre`

```
.app-c-govspeak_help__pre {
    padding: govuk-spacing(2);
    background-color: govuk-colour("light-grey");
    word-break: normal;
    word-wrap: normal;
    overflow: auto;
  }
}
```

FYI - I will be moving the files around so that we have the following CSS structure:

```
admin (for the new stuff)
  -> components
  -> views
  ...
admin_legacy (for all the old Bootstrap stuff)
  -> views
  -> helpers
  ...
```

I will be moving these things over in this card: https://trello.com/c/3UW25yLj/779-separate-the-asset-files-css-js-for-bootstrap-pages-vs-design-system-pages.

I hope this makes sense.

Reply:
nice one